src/components/text-input/index.tsx

```diff
@@ -0,0 +1,51 @@
+import Image from 'next/image';
+import { FC } from 'react';
+
+interface ITextInputProps {
+  iconSrc?: string;
+  iconAlt?: string;
+  placeholder: string;
+  value: string;
+  onInput: (value: string) => void;
+  onEnter?: () => void;
+}
+
+const TextInput: FC<ITextInputProps> = ({
+  iconSrc,
+  iconAlt,
+  placeholder,
+  value,
+  onInput,
+  onEnter,
+}) => {
```

Comment:
```suggestion
function TextInput(props: TextInputProps) {
const {
  iconSrc,
  iconAlt,
  placeholder,
  value,
  onInput,
  onEnter,
} = props

```
mình dùng style này nha em, cho tất cả component